homeworks/09-chokidar/src/dirWatcher.ts

```diff
@@ -0,0 +1,52 @@
+const fs = require("fs");
```

Comment:
Please use ES6 import syntax in all files

Reply:
Took me 5 minutes to convert everything to ES6 modules in your project and application is running fine.